install.sh

```diff
@@ -51,17 +51,22 @@ else
 	mount ${DISK}1 ${MOUNT_PATH}
 fi
 
+# chroot into target
 pacstrap ${MOUNT_PATH} base
-arch-chroot ${MOUNT_PATH} /bin/bash <<EOF
+arch-chroot ${MOUNT_PATH} /bin/bash
 echo "en_US.UTF-8 UTF-8" >> /etc/locale.gen
 locale-gen
 
+# adding multilib to pacman mirror list
 echo "
 [multilib]
 Include = /etc/pacman.d/mirrorlist
 " >> /etc/pacman.conf
 
+# system update
```

Comment:
well, not really an update, we need to do this to grab the multilib package database; perhaps the comment could say `update package datbases` ?